### PR TITLE
Added generation of NuGet symbol packages.

### PR DESCRIPTION
**Pull request by mario3001**
Changed build script to generate NuGet packages including symbols.

### CODE REVIEW

File: scripts/bbv.Common.build [hunk 1]
```diff
@@ -50,8 +50,9 @@
     <call target="copy_bin" />
     <call target="unittest" />
     <call target="revert_version" />
-    <call target="package" />
-    <call target="nuget" />
+    <call target="package" /> 
+    <!--<call target="nuget" />-->
```
Comment (ursenzler) [inline scripts/bbv.Common.build]: This line should not be commented, should it?

File: scripts/nuget.symbols.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+    <files>  
+      <file src="lib\**\*.dll" target="lib" />
```
Comment (ursenzler) [inline scripts/nuget.symbols.txt:2]: I don't get why the folder 'lib' is used. Where does this folder come from?
What am I missing?

File: scripts/bbv.Common.build (hunk 1 above)
Comment (mario3001) [inline scripts/bbv.Common.build]: You are right. I missed to remove the old NuGet code without sylbol generation.

File: scripts/nuget.symbols.txt (hunk 2 above)
Comment (mario3001) [inline scripts/nuget.symbols.txt:2]: The are the compiled assemblies in the subfolder of each NuGet package. for example the nuspec package in the folder ...\bbv.Common\Source\binaries\Release\bbv.Common includes the libs from its subfolder ...\bbv.Common\Source\binaries\Release\bbv.Common\lib. Due to NuGet documentation the libs are also needed in the symbol package.

**Comment by ursenzler** [comment]
I merged this pull request directly into Appccelerate (the new name of bbv.Common)

**Pull request closed without merging**